README.md

```diff
@@ -157,7 +157,7 @@ Enry's language detection has been compared with Linguist's language detection.
 
  Following results were obtained:
 
-![histogram](https://raw.githubusercontent.com/src-c/enry/master/benchmarks/histogram/distribution.jpg)
+![histogram](https://raw.githubusercontent.com/src-c/enry/master/benchmarks/histogram/distribution.png)
```

Comment:
This url is not working for me either. Why not just [histogram](benchmarks/histogram/distribution.png)?